readme.txt

```diff
@@ -181,26 +181,22 @@ INTERESTED IN DEVELOPMENT?
 
 = 4.0.0 - 2020-03-03 =
 
-* Add - Add woocommerce_admin_process_variation_object hook before $variation->save(), pass WC_Product_Variation object to hooks in the variation metabox. #24929
```

Comment:
We don't use `Add` in our changelogs, also those are dev related changes, so should use `Dev` prefix.